server/src/main/java/org/apache/cassandra/sidecar/coordination/ClusterLeaseClaimTask.java

```diff
@@ -144,7 +147,13 @@ public ScheduleDecision scheduleDecision()
     @Override
     public DurationSpec initialDelay()
     {
-        return periodicTaskConfiguration.initialDelay();
+        // Return a randomized delay to introduce jitter among all the instances participating
+        // in the lease claim process
+        MillisecondBoundConfiguration initialDelay = periodicTaskConfiguration.initialDelay();
+        long millis = initialDelay.toMillis();
+        return millis <= 0
+               ? initialDelay
+               : new MillisecondBoundConfiguration(ThreadLocalRandom.current().nextLong(millis), TimeUnit.MILLISECONDS);
```

Comment:
How can `initialDelay` be configured negative? 
How about adding a lower bound to the randomized delay? I am thinking of 1 second.

Reply:
it cannot be negative, it's restricted by `DurationSpec` however `ThreadLocalRandom.current().nextLong` throws when you provide it a value that is 0

Reply:
I could have done `==` but I think it's fine to leave a `<=` what do you think?

Reply:
Thinking it more. I think the current `RAND(initialDelay)` changes the semantics of the initial delay. It is almost guaranteed that the actual value is not the `initialDelay`. It could cause surprise. 
Would it make sense to split it into 2 parts, `minInitialDelay` and `maxInitialDelay`? The actual value is randomly picked that in the value range.

Reply:
I think that's fine, but I would probably like to address it as part of a different effort. What do you think ?  In that case we'd also need to address the randomized initial delay in `org.apache.cassandra.sidecar.datahub.SchemaReportingTask#initialDelay`